File: cfgs/java_cfgs_foo/POSITION_OF_RIGHTMOST_SET_BIT_1.java
Nodes
1;special;;;
2;statement;int position = 1;14;
3;statement;int m = 1;15;
4;conditional;(foo_1 & foo_2) == 0;16;
5;statement;m = foo_1 << 1;17;
6;statement;position++;18;
7;statement;return foo_1;20;
8;exit;;;
Edges
0;2;
1;3;
2;4;
3;7;5;
4;6;
5;4;
6;8;
7;

File: cfgs/pythoncfgs/POSITION_OF_RIGHTMOST_SET_BIT_1.py
Nodes
1;statement;foo_1 = 1;8;
2;statement;foo_1 = 1;9;
3;conditional;not foo_1 & foo_2;10;
4;statement;return foo_1;13;
5;statement;foo_1 = foo_2 << 1;11;
6;statement;foo_1 += 1;12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;6;
5;3;
6;